From: Christoph Hellwig <hch@lst.de>
Date: Mon, 23 Jan 2017 14:31:09 +0100
Subject: [PATCH] block: don't assign cmd_flags in __blk_rq_prep_clone
Git-commit: fb045ca25cc7b6d46368ab8221774489c2a81648
Patch-Mainline: v4.11
References: bsc#1088087

These days we have the proper flags set since request allocation time.

Signed-off-by: Christoph Hellwig <hch@lst.de>
Acked-by: Martin K. Petersen <martin.petersen@oracle.com>
Reviewed-by: Hannes Reinecke <hare@suse.com>
Signed-off-by: Jens Axboe <axboe@fb.com>
---
 block/blk-core.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/block/blk-core.c b/block/blk-core.c
index b565b6b19c1c..68a5199de7fe 100644
--- a/block/blk-core.c
+++ b/block/blk-core.c
@@ -3043,7 +3043,6 @@ EXPORT_SYMBOL_GPL(blk_rq_unprep_clone);
 static void __blk_rq_prep_clone(struct request *dst, struct request *src)
 {
 	dst->cpu = src->cpu;
-	dst->cmd_flags = src->cmd_flags | REQ_NOMERGE;
 	dst->cmd_type = src->cmd_type;
 	dst->__sector = blk_rq_pos(src);
 	dst->__data_len = blk_rq_bytes(src);
-- 
2.12.3

